packages/terra-tabs/src/common-tabs/_Tab.jsx

```diff
@@ -168,6 +178,43 @@ const Tab = ({
   attributes['aria-selected'] = isSelected;
   attributes.style = { zIndex };
 
+  // TBD: Add Translations
+  const onFocusResponse = 'Press Enter to activate a tab, or Press space bar to start a drag. When dragging you can use the arrow keys to move the item around and escape to cancel. Ensure your screen reader is in focus mode or forms mode';
+  const responseId = `terra-tab-pane-response=${uuidv4()}`;
+
+  if (isDraggable) {
+    return (
+      <Draggable key={id} draggableId={id} index={index}>
+        {(provided) => (
+          <div
+            {...attributes}
+            ref={provided.innerRef}
+            {...provided.draggableProps}
+            {...provided.dragHandleProps}
+            id={id}
+            aria-controls={associatedPanelId}
+            role="tab"
+            className={variant === 'framework' ? paneClassNames : tabClassNames}
+            title={label}
+            aria-describedby={responseId}
+            tabIndex={isSelected ? 0 : -1}
+            data-terra-tabs-show-focus-styles
+            data-terra-tab-draggable
+          >
+            <div>
+
+              <IconKnurling className={cx('icon-knurling')} />
+              <VisuallyHiddenText aria-hidden id={responseId} text={onFocusResponse} />
+              {customDisplay}
+              {customDisplay ? null : icon}
+              {customDisplay || isIconOnly ? null : <span className={cx('label')}>{label}</span>}
```

Comment:
nit: Can we update the above condition like below to avoid null 

```suggestion
        {customDisplay || icon}
        {(!customDisplay || !isIconOnly) && <span className={cx('label')}>{label}</span>}
```